### PR TITLE
Editorial pass

### PR DESCRIPTION
**Pull request by anssiko**
I did a full editorial pass on this document. Given it was in a great shape already, this PR contains just some grammar checks, nitpicks, and cross-referencing improvements.

(I did not adjust the line length in the source as to not introduce noise.)

PTAL @kenchris et al.

***
[Preview](https://s3.amazonaws.com/pr-preview/w3c/motion-sensors/editorial.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/motion-sensors/2b9929b...9ed78b7.html)

### CODE REVIEW

**Comment by kenchris** [comment]
I looked quickly and it looks good.

File: index.bs [hunk 1]
```diff
@@ -438,15 +438,15 @@ results and is easy to implement in hardware, this is a common solution.
...
+A <dfn>complementary filter</dfn> can be thought of as a [=low-pass filter=] and [=high-pass filter=] in one,
 complementing the [=gyroscope=] values with the [=accelerometer=] values:
 
 θ<sub>n</sub> = α × (θ<sub>n-1</sub> + ω × ∂t) + (1.0 - α) × a
 
 With α being the weight constant, a the acceleration from [=accelerometer=], ω the angular
 velocity from [=gyroscope=] and ∂t being the time between measurements.
 
-A common value for α is 0.98, which means that 98% of the weight lays on the [=gyroscope=]
+A common value for &alpha; is 0.98, which means that 98% of the weight lays on the [=gyroscope=]
```
Comment (anssiko) [inline index.bs]: @kenchris This is now an html entity. Can we do a global change to html entitities in the spec, or do I need to ditch TextMate :P

Comment (kenchris) [reply index.bs]: Ditch TextMate :) not all have nice names like &alpha;

**Comment by anssiko** [comment]
@kenchris, I resolved the merge conflict introduced by 2b9929b1a6308fb51fb08ae4dd56e46f31bd7c5e

Please merge.

**Comment by anssiko** [comment]
Thanks!

**Pull request merged**